extensions/amp-list/0.1/amp-list.js

```diff
@@ -567,7 +567,12 @@ export class AmpList extends AMP.BaseElement {
     const isSSR = this.ssrTemplateHelper_.isSupported();
     let renderPromise = this.ssrTemplateHelper_.renderTemplate(
         this.element, current.data)
-        .then(result => this.updateBindings_(result, current.append))
+        // For SSR, the result will be the container node that contains the
+        // list items. Just pass in the list items when updating the bindings
+        // and rendering else the sanitizer will strip out the class attribute
+        // from the container.
+        .then(result => this.updateBindings_(
+            isSSR ? toArray(result.childNodes) : result, current.append))
```

Comment:
What value of `class` attribute is being stripped? DOMPurify itself doesn't strip the `class` attribute, but maybe we are sanitizing the specific value.

Reply:
Right, it is actually behaving as it should, stripping the following amp reserved css.

    <div role="list" class="**i-amphtml-fill-content i-amphtml-replaced-content**">
       <div class="product" role="listitem">Apple</div>
       ...
    </div>